package.json

```diff
@@ -27,7 +27,7 @@
     "grunt-todo": "0.5.0",
     "jasmine-ajax": "3.2.0",
     "jasmine-core": "2.5.2",
-    "karma": "1.4.0",
+    "karma": "1.4.1",
```

Comment:
Karma 1.4 is evil! Please don't merge it: https://github.com/karma-runner/karma/issues/2552